src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/NomisTimeServedLicence.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "nomis_time_served_licences")
+class NomisTimeServedLicence(
+  @Column(name = "noms_id", length = 7, nullable = false)
+  val nomsId: String,
```

Comment:
nickpick: Nomis Is is really an internal legacy system and should be encapsulated,.. we should used prison number,.. because when we don't have Nomis in the future it's has no meaning.